airbyte-integrations/connectors/destination-pinecone/metadata.yaml

```diff
@@ -13,7 +13,7 @@ data:
   connectorSubtype: vectorstore
   connectorType: destination
   definitionId: 3d2b6f84-7f0d-4e3f-a5e5-7c7d4b50eabd
-  dockerImageTag: 0.1.26
+  dockerImageTag: 1.0.0-rc.1
```

Comment:
@natikgadzhi @clnoll I'm a bit confused, do we want to release an RC with a breaking change? 
If we merge it in its current form I'm afraid that pinned connections will fail because of the new required fields in the config.